mimic/rest/maas_api.py

```diff
@@ -622,14 +624,22 @@ def view_agent_host_info(self, request, tenant_id):
         """
         No agent monitoring. For now, always return 400.
         """
-        request.setResponseCode(400)
-        return json.dumps({
-            "type": "agentDoesNotExist",
-            "code": 400,
-            "message": "Agent does not exist",
-            "details": "Agent XYZ does not exist.",
-            "txnId": ".fake.mimic.transaction.id.c-1111111.ts-123444444.v-12344frf"
-        })
+        entity_id = request.args['entityId'][0].strip()
+        for e in self._entity_cache_for_tenant(tenant_id).entities_list:
+            if e['id'] == entity_id:
+                agent_id = e['agent_id']
+        self._entity_cache_for_tenant(tenant_id).agenthostinfo_querycount[entity_id] += 1
+        if self._entity_cache_for_tenant(tenant_id).agenthostinfo_querycount[entity_id] < 5:
```

Comment:
No, I really want it to fail 4 times. So there is no time limit. The 4 times can happen in less than a second or be minutes apart. It's just important that it's some known number of attempts.